u4-databases/features/uni-controller.js

```diff
@@ -0,0 +1,54 @@
+const { db } = require('../_services/firebase-initialized')
+
+const iterationList = async (req, res) => {
+  const query = db.collection('iterations').get()
+  const iterations = (await query).docs.map(doc => doc.data())
+
+  res.render('uni-iteration-List', { iterations })
+}
+
+const iterationDetails = async (req, res) => {
+  const code = req.params.code
+  const query = db.collection('iterations').doc(code).get()
+  const iteration = (await query).data()
+  console.log(iteration)
```

Comment:
คุณควรลบ `console.log` นี้ออกหลังจากแก้ไขจุดบกพร่องเสร็จแล้ว
You should remove this `console.log` after you finish debugging.